### PR TITLE
[WIP] Series Label created with tags

### PR DESCRIPTION
**Pull request by utkarshcmu**
Resolving #7309

### CODE REVIEW

**Comment by torkelo** [comment]
Looks good! It does not seem to handle alias patterns yet? is that why there is a [WIP] status?

**Comment by utkarshcmu** [comment]
@torkelo You are absolutely right!

**Comment by utkarshcmu** [comment]
@bergquist Can you look into this please whenever you have a few minutes? I am creating the label with the tags now but still this is not handling alias patterns in the Alerting message. If I am missing anything, please let me know so that I can implement that we can merge this PR. Thanks a lot for your help!

File: pkg/tsdb/opentsdb/opentsdb.go [hunk 1]
```diff
@@ -218,3 +216,15 @@ func (e *OpenTsdbExecutor) buildMetric(query *tsdb.Query) map[string]interface{}
 
 	return metric
 }
+
+func (e *OpenTsdbExecutor) createSeriesLabel(res OpenTsdbResponse) string {
```
Comment (torkelo) [inline pkg/tsdb/opentsdb/opentsdb.go]: dont you need to handle the alias pattern interpolation here? This is done in the frontend code here, https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/opentsdb/datasource.js#L348

**Comment by vijaykansal3** [comment]
Created [#12336](https://github.com/grafana/grafana/pull/12336) to fix this completely.

@utkarshcmu @torkelo Please have a look at the [#PR](https://github.com/grafana/grafana/pull/12336)

**Pull request merged**